godotproject/Player.gd

```diff
@@ -229,7 +219,8 @@ func _draw():
 	elif $PlayerTongue.swinging:
 		tongue_target_global = swing_pivot_position
 	
-	$Line2D.remove_point(1)
+	if $Line2D.get_point_count() > 1:
+		$Line2D.remove_point(1)
```

Comment:
This fixes the error that is printed every frame - we only remove the point if it exists. [FGTS-94]